Cargo.toml

```diff
@@ -16,7 +16,7 @@ prost-codec = ["prost", "prost-derive", "bytes"]
 regenerate = []
 
 [dependencies]
-protobuf = { version = "2", optional = true }
+protobuf = { version = "=2.0.6", optional = true }
```

Comment:
Why change this version?

Reply:
Without '=' it will use latest '2.x', which is '2.8.0' now and has compatibility problem.